features/notifications/notifications_controller.rb

```diff
@@ -0,0 +1,91 @@
+require_relative "../../shared/authenticated_controller_base"
+require_relative "../../services/services"
+require_relative "../../taskqueue/task_queue"
+require "pathname"
+
+module FastlaneCI
+  # CRUD controller for handling and displaying notifications to user
+  class NotificationsController < AuthenticatedControllerBase
+    HOME = "/notifications"
+
+    # Instantiates a new `Notification Controller` with a task queue to process
+    # requests
+    #
+    # @param  [Sinatra::Base] app
+    # @return [nil]
+    def initialize(app = nil)
+      @task_queue = TaskQueue::TaskQueue.new(name: "notifications")
```

Comment:
I know we're not super consistent on this, but maybe we should standardize the order in which we call `super()`
Generally speaking, I would opt for calling `super()` before changing any state in the current initializer.